packages/docs/src/components/sidebar/navigationRoutes.ts

```diff
@@ -91,6 +91,13 @@ export const navigationRoutes: NavigationRoute[] = [
     },
```

Comment:
This command didn't work for me: `node ./node_modules/@vuestic/tailwind/dist/sync.cjs`.

Also in docs we use tsx, which users might not have.

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/5394573/210325076-fe9ed390-258e-4cae-af9b-21e48339bdae.png">

Reply:
Fixed. Wasn't proceeded situation if no tailwind.config file was found. 

Script reworked to work via `node`.